packages/virtualized-list/src/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { Component, For, JSX, createSignal } from "solid-js";
+
+type VirtualizedListProps<T> = {
+  class?: string;
+  items: T[];
+  overscanCount?: number;
+  renderRow: (row: T) => JSX.Element;
```

Comment:
Instead of renderRow, consider using a FlowComponent and children for the JSX conversion like our List primitive does.

Reply:
I'm not familiar with how to implement something like that, but I like that API much more, so I'll try. I'll refer to the List primitive, but know that it might take a little while.

Reply:
It's just a helper type solid provides.